service/internal/cloudconfig/tccpn.go

```diff
@@ -348,6 +348,7 @@ func (t *TCCPN) newTemplate(ctx context.Context, obj interface{}, mapping hamast
 		params.DisableIngressControllerService = true
 		params.DockerhubToken = t.config.DockerhubToken
 		params.EnableAWSCNI = true
+		params.EnableCSIMigrationAWS = true
```

Comment:
this will enable the transition, so basically if users create a new PVC with the old storage class it will be migrated to CSI driver handling it.